### PR TITLE
Improve `enable` and `disable` commands + Update `yiisoft/files` dependency

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | -

Already enabled/disabled packages are taken into account separately:
image

### CODE REVIEW

File: src/App/Command/DisableCommand.php [hunk 1]
```diff
@@ -48,11 +48,16 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             $disablePackageIds = array_unique(explode(',', $commaSeparatedPackageIds));
         }
 
+        $alreadyDisabledPackages = [];
         $disabledPackages = [];
         foreach ($disablePackageIds as $packageId) {
             if (isset($packages[$packageId])) {
```
Comment (arogachev) [inline src/App/Command/DisableCommand.php:54]: I'd used an early return here:

```php
if (!isset($packages[$packageId])) {
    continue;
}

// The rest of the code

File: src/App/Command/DisableCommand.php [hunk 2]
```diff
@@ -65,10 +70,15 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         fwrite($handle, '];' . "\n");
         fclose($handle);
 
-        if (!empty($disabledPackages)) {
-            $io->success("Disabled packages:\n — " . implode("\n — ", $disabledPackages));
-        } else {
+        if (empty($alreadyDisabledPackages) && empty($disabledPackages)) {
             $io->info('Packages not found.');
```
Comment (arogachev) [inline src/App/Command/DisableCommand.php:74]: Similar to https://github.com/yiisoft/yii-dev-tool/pull/219/files#r947529868.

File: src/App/Command/EnableCommand.php [hunk 3]
```diff
@@ -48,11 +48,16 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             $enablePackageIds = array_unique(explode(',', $commaSeparatedPackageIds));
         }
 
+        $alreadyEnabledPackages = [];
         $enabledPackages = [];
         foreach ($enablePackageIds as $packageId) {
             if (isset($packages[$packageId])) {
-                $packages[$packageId] = true;
-                $enabledPackages[] = $packageId;
+                if ($packages[$packageId]) {
```
Comment (arogachev) [inline src/App/Command/EnableCommand.php:55]: Similar to https://github.com/yiisoft/yii-dev-tool/pull/219/files#r947529868.

File: src/App/Command/EnableCommand.php [hunk 4]
```diff
@@ -65,10 +70,15 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         fwrite($handle, '];' . "\n");
         fclose($handle);
 
-        if (!empty($enabledPackages)) {
-            $io->success("Enabled packages:\n — " . implode("\n — ", $enabledPackages));
-        } else {
+        if (empty($alreadyEnabledPackages) && empty($enabledPackages)) {
             $io->info('Packages not found.');
```
Comment (arogachev) [inline src/App/Command/EnableCommand.php:74]: Similar to https://github.com/yiisoft/yii-dev-tool/pull/219/files#r947529868

**Review by arogachev — CHANGES_REQUESTED** [review]
:+1:

**Comment by arogachev** [comment]
Could you take a screenshot of how it was before these changes?

**Comment by arogachev** [comment]
Forgot to mention that these comments are code style related meaning they are subjective / optional.

**Comment by vjik** [comment]
> Could you take a screenshot of how it was before these changes?

Before all enabled/disabled packages was in single list.

File: src/App/Command/EnableCommand.php (hunk 4 above)
Comment (vjik) [reply src/App/Command/EnableCommand.php:76]: Done.

File: src/App/Command/EnableCommand.php (hunk 3 above)
Comment (vjik) [reply src/App/Command/EnableCommand.php]: Done.

File: src/App/Command/DisableCommand.php (hunk 2 above)
Comment (vjik) [reply src/App/Command/DisableCommand.php:76]: Done.

File: src/App/Command/DisableCommand.php (hunk 1 above)
Comment (vjik) [reply src/App/Command/DisableCommand.php]: Done.

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**